Userland/Shell/Builtin.cpp

```diff
@@ -39,6 +39,43 @@ int Shell::builtin_dump(int argc, char const** argv)
     return 0;
 }
 
+int Shell::builtin_where(int argc, char const** argv)
+{
+    bool at_least_one_succeded = false;
+    for (ssize_t i = 1; i < argc; i++) {
+        auto current_command = DeprecatedString(argv[i]);
+        auto alias = m_aliases.get(current_command);
+        if (alias.has_value()) {
```

Comment:
Just thinking out loud, does zsh's where have flags that say "only print yay kind of results"? Like `--only-paths` to skip aliases and builtins?

Reply:
Thanks for the feedback. Done.